### ISSUE TITLE
Looking for a chair for upcoming performance meetings

### ISSUE THREAD

**Issue by anonrig** [nodejs/performance#143]
Hi everybody,

Due to personal reasons & developments, I'm having hard time chairing the performance meetings. I'd like to publicly ask if there is a Node.js collaborator who'd like to chair the upcoming performance meetings?

Ideally, this person will get access to Zoom & Youtube accounts of Node.js (through GitHub.com/nodejs/admin) and start & live-stream meetings, and take meeting notes.

We could also potentially explore to have a `co-chair` in case the new chair is unavailable in the future.

cc @nodejs/performance @nodejs/tsc

**Comment by Uzlopak** [comment]
I nominate myself. 

I also nominate @H4ad and @rluvaton

**Comment by H4ad** [comment]
I usually have a regularly scheduled meeting on my work at the same time as the performance team meeting, so even if I want to be the `co-chair` I don't know if I can join the meetings.

**Comment by UlisesGascon** [comment]
I served as a `co-chair` for the performance meetings in the past, but unfortunately, the current time slot doesn't align well with my schedule. However, I am more than happy to help onboarding new co-chair(s) and explain all the details regarding the zoom settings and streaming details 😄

**Comment by Qard** [comment]
Seems to be a universal experience that the time slot is not great. Same sentiment here. 🙈

**Comment by RafaelGSS** [comment]
The same applies to me. I'm wondering if we could integrate the nodejs/diagnostics with nodejs/performance meetings.

Both meetings don't fit in my calendar, but we could find a better time and host both topics.

**Comment by H4ad** [comment]
@RafaelGSS I liked the idea, currently I don't think the topics for the performance will take the whole hour, if we join both topics we maybe can use the whole time of the meeting and cover all the topics with more people.

**Comment by Qard** [comment]
Makes sense to me, especially given most diag wg sessions have mainly been about async_hooks perf for awhile now. 😅

**Comment by iicolin** [comment]
That sounds like a big responsibility, and it’s great that you’re looking for a collaborator to ensure the meetings continue smoothly. Hopefully, someone from the Node.js community steps up to take on the role! Having a co-chair is also a smart idea for backup support.

And speaking of chairs, if you're spending long hours in meetings, [chair covers in Australia](https://ozsupermart.com.au/) can be a lifesaver—keeping your seating comfy and stylish while you tackle all those performance discussions! 😆 Hope you find the right person soon!
